### ISSUE TITLE
Change request - Custom locator types Console Output statement to log4j logger output

### ISSUE THREAD

**Issue by kulin24** [qmetry/qaf#369]
### Steps To Reproduce
    1. Use custom locator strategy like accessibility id for any element.
    2. Execute the script and check the console output.
   

### Expected behavior
Instead of a console output statement, if the logger is used then the users can disable the logs if they change the log level to error.

### Actual behavior
Currently, if the users are trying to disable the logs by changing the log level to error in the log4J configurations, then also the following line is seen in the console output - 

No class registerd for stretegyaccessibility id. Will use 'accessibility id' as custom stretegy

### Is the issue reproducible on runner?
Yes

- [x] QAS
- [x] Maven
- [ ] Gradle
- [] Ant
- [x] Eclipse

### Test case sample
> Please, share the test case (as small as possible) which shows the issue
Just using any test case with accessibility id as the locator strategy, we should be able to reproduce the issue.

This issue is raised on behalf of a Quantum user who has requested for this console output statement to logger logs so that it can be disabled using log4J properties when they want to suppress the unwanted logs.

System.out.println("No class registerd for stretegy" + s + ". Will use '" + s + "' as custom stretegy");

Also, there are minor spelling mistakes in the current console output statement for two words - "registerd" & "stretegy"
So they can be changed to - "registered" & "strategy"

**Comment by cjayswal** [comment]
This request looks reasonable. Please add reference to this issue in pull request.
